bin/node/bench/src/trie.rs

```diff
@@ -204,6 +207,138 @@ impl core::Benchmark for TrieBenchmark {
 	}
 }
 
+pub struct TrieWriteBenchmarkDescription {
+	pub database_size: DatabaseSize,
+}
+
+impl core::BenchmarkDescription for TrieWriteBenchmarkDescription {
+	fn path(&self) -> Path {
+		let mut path = Path::new(&["trie", "write"]);
+		path.push(&format!("{}", self.database_size));
+		path
+	}
+
+	fn setup(self: Box<Self>) -> Box<dyn core::Benchmark> {
+		let mut database = TempDatabase::new();
+
+		// TODO: make seedable
```

Comment:
only merge TODOs with link to issue on github. Please either create the issue and link it here or remove the todo.